Source/DafnyCore/Verifier/ProofObligationDescription.cs

```diff
@@ -290,7 +308,7 @@ public class PreconditionSatisfied : ProofObligationDescription {
   }
 }
 
-public class AssertStatement : ProofObligationDescription {
+public class AssertStatement : ProofObligationDescriptionNotInDafny {
```

Comment:
Why don't we implement `GetAssertedExpr` for this one right away, since it should be readily available?

Reply:
The assertion is made of Boogie expressions... It is used in ensures and requires as well. This will require more work that I already started in a separate branch.

Reply:
Ah, okay, let's leave it out for now, then. :)